docs/docs/tutorials.md

```diff
@@ -50,21 +50,6 @@ title: Tutorials and Explainers
 
 - Learn how to perform authentication with an OIDC Application in Okta and NextAuth.js.
 
-## Fullstack
```

Comment:
Can we keep the header? The tutorial below belongs in this category too it looks like

Reply:
Oops! Yes, I've made the update now :))